NetworkingTests/Tests/EntityRepositorySpec.swift

```diff
@@ -154,6 +191,23 @@ class EntityRepositorySpec: QuickSpec {
             
         }
         
+        describe("#fetchCustomFailingEntityMishandlingError") {
+            
+            it("fetches a single entity from JSON file and fails with a custom error mishandled") { waitUntil { done in
+                repository.fetchCustomFailingEntityMishandlingError().startWithResult {
+                    switch $0 {
+                    case .success: fail()
+                    case .failure(let error):
+                        switch error {
+                        case .requestError: done()
```

Comment:
This is OK.

This test validates that when a request fails with an error not handled in the map provided to map, it just returns the same request error.